src/Generator/Passes/CleanCommentsPass.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using CppSharp.AST;
+
+namespace CppSharp.Passes
+{
+    public class CleanCommentsPass : TranslationUnitPass
+    { 
+        public override bool VisitDeclaration(Declaration decl)
+        {
+            if (decl.Comment != null)
+            {
+                var summaryPara = (ParagraphComment)decl.Comment.FullComment.Blocks[0];
```

Comment:
@tritao this needs rewriting rather than fixing. It had become too late so I advised @ktopouzi to just continue tomorrow.